deep_translator/google_trans.py

```diff
@@ -62,6 +84,8 @@ def _map_language_to_code(self, *languages):
                 yield language
             elif language in self._languages.keys():
                 yield self._languages[language]
+            elif (self.is_secondary(language)!=0):
```

Comment:
`elif self.is_secondary(language) `would be sufficient in this case. You do not need to check more in python.

However, if you check this in` is_language_supported`, then you don't have to check this here again

Reply:
done